src/Evolution/Systems/GeneralizedHarmonic/Tags.hpp

```diff
@@ -166,6 +166,52 @@ struct EvolvedFieldsFromCharacteristicFields : db::SimpleTag {
     return "EvolvedFieldsFromCharacteristicFields";
   }
 };
+
+// @{
+/*!
+ * \brief Tags corresponding to various constraints of the generalized
+ * harmonic system, and their diagnostically useful combinations.
+ * \details For details on how these are defined and computed, see
+ * `GaugeConstraintCompute`, `FConstraintCompute`, `TwoIndexConstraintCompute`,
+ * `ThreeIndexConstraintCompute`, `FourIndexConstraintCompute`, and
+ * `ConstraintEnergyCompute` respectively
+ */
+template <size_t SpatialDim, typename Frame>
+struct GaugeConstraint : db::SimpleTag {
+  using type = tnsr::a<DataVector, SpatialDim, Frame>;
+  static std::string name() noexcept { return "GaugeConstraint"; }
+};
+
+template <size_t SpatialDim, typename Frame>
+struct FConstraint : db::SimpleTag {
```

Comment:
Hmm, I'm not getting the documentation showing up on anything but the first tag (`GaugeConstraint`). It's also not obvious to me why that is

Reply:
Documentation shows up fine for all constraint tags for me. See this screenshot:

![Screenshot from 2019-05-22 13-50-58](https://user-images.githubusercontent.com/7142969/58196716-ec217680-7c98-11e9-8226-33da4ba5d207.png)

Reply:
Hmm, this seems to be common for struct documentations, and plausibly has something to do with global Doxy settings.

For e.g., 3 structs are supposed to share documentation in [`src/ApparentHorizons/ComputeItems.hpp`](https://github.com/sxs-collaboration/spectre/blob/develop/src/ApparentHorizons/ComputeItems.hpp#L25), but only for the first one does the documentation actually show up ([this one](https://spectre-code.org/structah_1_1tags_1_1inversespatialmetriccompute)), while for the other two (see [this](https://spectre-code.org/structah_1_1tags_1_1extrinsiccurvaturecompute) and [this](https://spectre-code.org/structah_1_1tags_1_1spatialchristoffelsecondkindcompute)) it doesn't.

On the other hand, when grouping functions together, doxygen works as expected!! For e.g. we have the overloaded [`characteristic_speeds`](https://spectre-code.org/namespacegrmhd_1_1valenciadivclean#a5c22ceb9a14eae9cca29b9aa108b8ca9) functions of GrMhd system, or functions that compute the [`unnormalized_face_normal`](https://spectre-code.org/group__computationaldomaingroup#ga417a2e1563c807ebf515d4e9fa04d425).

Reply:
As we discussed, I've resorted to `@copydoc`.